text/0000-informational-commands.md

```diff
@@ -0,0 +1,525 @@
+
+- "human" mode, the default if the context is a user-facing terminal; also invokable with `--print=human` in any context.
+
+- "plain" mode, the default if the context is not a user-facing terminal (e.g. when piped into another command); also invokable with `--print=plain` in any context. A simple plain text format which prints a line per runtime, packager, or tool, with space-separated output on each line.
+
+    - runtimes:
+
+        ```
+        runtime node@<version>
+        ```
+
+    - packagers:
+
+        ```
+        packager (yarn|npm)@<version>
+        ```
+
+    - tools: 
+
+        ```
+        tool <tool name>@<tool version> [node@<version>] [<yarn|npm>@<version>] [(default|<project path>)]
+        ```
+
+Having these two modes should make it easy to add a JSON mode later (`--print=json`) if that proves desirable.
+
+## CLI Command Variants
+
+### `volta list` (no flags)
+
+#### Human
+
+The format is:
+
+```sh
+$ volta list
+⚡️ Currently active toolchain:
+
+    Node: version [(default|<project path>)]
+    <packager>: <built-in|version> [(default|<project path>)]]
+    Tool binaries available:
+        <comma separated list|NONE>
+
+See options for more detailed reports by running `volta list --help`.
+```
+
+<details><summary>Outside a project</summary>
+
+```sh
+$ volta list
+⚡️ Currently active toolchain:
+
+    Node: v8.16.0 (default)
+    Yarn: v1.12.3 (default)
+    Tool binaries available:
+        create-react-app, ember, tsc, tsserver, yarn-deduplicate
+
+See options for more detailed reports by running `volta list --help`.
+```
+
+</details>
+
+<details><summary>In the `node-only` project</summary>
+
+<b>Note:</b> this assumes the implementation of a fix for [volta-cli/volta#436](https://github.com/volta-cli/volta/issues/436).
+
+```sh
+$ volta list
+⚡️ Currently active toolchain:
+
+    Node: v8.16.0 (from `~/node-only/package.json`)
+    Yarn: v1.12.3 (default)
+    Tool binaries available:
+        create-react-app, ember, tsc, tsserver, yarn-deduplicate
+
+See options for more detailed reports by running `volta list --help`.
+```
+
+</details>
+
+<details><summary>In the <code>node-and-yarn</code> project</summary>
+
+```sh
+$ volta list
+⚡️ Currently active toolchain:
+
+    Node: v12.2.0 (from `~/node-and-yarn/package.json`)
+    Yarn: v1.16.0 (from `~/node-and-yarn/package.json`)
+    Tool binaries available:
+        create-react-app, ember, tsc, tsserver, yarn-deduplicate
+
+See options for more detailed reports by running `volta list --help`.
+```
+    
+</details>
+
+#### Plain
+
+The format is:
+
+```sh
+$ volta list
+runtime node@<version> (default|<project path>)
+packager <npm|yarn>@<version> (built-in|default|<project path>)
+```
+
+<details><summary>Outside a project</summary>
+
+```sh
+$ volta list
+runtime node@v10.15.3 (default)
+packager yarn@v1.12.3 (default)
```

Comment:
I'm unsure what we should actually output here: if the user has `install`-ed Yarn, presumably we should print it. Should we also always print the built-in `npm`? I currently lean toward yes; that would then extend to the `human` format as well. Thoughts?